package.nls.json

```
@@ -446,5 +446,6 @@
     "DataScience.trimmedOutput": "Output was trimmed for performance reasons.\nTo see the full output set the setting \"python.dataScience.textOutputLimit\" to 0.",
     "DataScience.connectingToJupyterUri": "Connecting to Jupyter server at {0}",
     "DataScience.createdNewNotebook": "{0}: Creating new notebook ",
-    "DataScience.createdNewKernel": "{0}: Kernel started: {1}"
+    "DataScience.createdNewKernel": "{0}: Kernel started: {1}",
+    "OutdatedDebugger.updateDebuggerMessage": "We noticed you are attaching to ptvsd, which will be deprecated on May 1st, 2020. Please update to the new debugger, debugpy."
```

```suggestion
    "OutdatedDebugger.updateDebuggerMessage": "We noticed you are attaching to ptvsd, which will be deprecated on May 1st, 2020. Please update to the new debugger, [debugpy](https://aka.ms/migrateToDebugpy)."
```

Messages in VSC now support markdown links. Better than a more info button, or something on top of that.

---

Note, some tools (other extensions) launch python programs then attach the debugger.
I.e. user is not manually attaching. I think its better to rephrase to to `Python debugger(ptvsd)` or `ptvsd (Python Debugger)`, or similar.

---

E.g. tools/extensions built by MS included..

---

@luabud Let me know if the message changes are OK. I agree with Don we need to be clearer here.